core/fokia/cluster_creator.py

```diff
@@ -4,89 +4,109 @@
 import inspect
 from fokia.provisioner import Provisioner
 from fokia.ansible_manager import Manager
+# script_path = os.path.dirname(os.path.abspath(inspect.getfile(inspect.currentframe())))
+script_path = '/var/www/okeanos-LoD/core/fokia'
```

Comment:
This is the path of the repo on the service VM, when it is deployed via ansible.